.github/workflows/ci-prq-reports.yml

```diff
@@ -0,0 +1,76 @@
+name: PR Quality Reports
+on:
+  workflow_run:
+    workflows: [ "PR Quality" ]
+    types:
+      - completed
+jobs:
+  checkstyle:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        java: [ 8, 11 ]
+    steps:
+      - name: Download Artifacts
+        # v2.10.0
+        uses: dawidd6/action-download-artifact@5b72739630efa11b8d81784b89adf5246a5fc475
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          workflow: ${{ github.event.workflow_run.workflow_id }}
+          commit: ${{ github.event.workflow_run.head_commit.id }}
+          name: checkstyle-results-${{ matrix.java }}
+      - name: Publish Checkstyle Report
+        # v1.2
+        uses: jwgmeligmeyling/checkstyle-github-action@50292990e18466f2c5d95d04ff5fab931254fa5f
+        with:
+          name: Checkstyle Report JDK ${{ matrix.java }}
+          path: '**/build/reports/checkstyle/*.xml'
+          token: ${{ secrets.GITHUB_TOKEN }}
+          # TODO https://github.com/jwgmeligmeyling/checkstyle-github-action/issues/6
```

Comment:
note we will not get the quality reports or GitHub annotations until the Github Actions resolve these comments.